src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -72,11 +70,14 @@ public IntakeSubsystem() {
 		motor1.setIdleMode(IdleMode.kBrake);
 		motor2.setIdleMode(IdleMode.kBrake);
 
+		// need cause motors run opposite direciton.
+		motor1.setInverted(true);
```

Comment:
Should both be inverted or just one in order for them to run in the opposite direction?

Reply:
To be clear, the comment refers to the motors running in opposite directions as before, not that they are being set to run in opposite directions as each other.